docker/build/Dockerfile

```diff
@@ -26,6 +26,7 @@ RUN \
 
 COPY run /usr/bin/
 COPY sudoers /etc/
+COPY bin/* /usr/bin/
```

Comment:
I do not see where this bin forlder come from ?

Reply:
From here: https://github.com/camptocamp/c2cgeoportal/tree/ci-doc/docker/build/bin